scripts/suse_functions.sh

```diff
@@ -0,0 +1,35 @@
+OBS_DIR=${OBS_DIR:-~/obs}
+source_dir="$(pwd)"
+package_name="$(basename $(pwd))"
+prepare_obs() {
+    project_name="$1"
+
+    tempdir="$(mktemp -d)"
+    echo "Packaging sources w/ tito"
+    tito build --srpm --test -o "$tempdir"
+
+    if [ ! -d ~/obs ]; then
+        mkdir ~/obs
+    fi
+    pushd ~/obs
+
+    if [ ! -d "$project_name" ]; then
+        osc co "$project_name"
+        pushd "$project_name"
+    else
+        pushd "$project_name"
+        echo "Ensuring $project_name is up-to-date."
+        osc update
+    fi
+
+    pushd "$package_name"
+    echo "Removing any existing sources"
+    osc rm *
```

Comment:
Okay.  That's fine then.  I just didn't want my home directory [disappearing accidentally](http://www.techrepublic.com/article/moving-steams-local-folder-deletes-all-user-files-on-linux/)